app/celer-g4/GlobalSetup.cc

```diff
@@ -134,30 +134,26 @@ void GlobalSetup::ReadInput(std::string const& filename)
     nlohmann::json::parse(infile).get_to(input_);
     CELER_ASSERT(input_);
 
+    options_->output_file = input_.output_file;
+    options_->offload_output_file = input_.offload_output_file;
+    // XXX where are geometry and physics file set?
```

Comment:
I guess we only want to set the geometry filename in the accel `SetupOptions` if we want to load the geometry from the GDML rather than import it from Geant4... maybe we need another input option to specify which to do? The physics output file I never added to the celer-g4 run input, so we'd need to add that as well.

Reply:
I think we can move the geometry file from `RunAction::BeginOfRunAction`, and I might as well add the physics output option while i'm at it.